components/fieldgroups/intent/demandbase-account-intent-details.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/demandbase-account-intent-details",
```

Comment:
Please change the $id to `https://ns.adobe.com/b2b/demandbase/mixins/demandbase-account-intent-details`